src/kemal/middleware/filters.cr

```diff
@@ -28,6 +33,13 @@ module Kemal::Middleware
     private def radix_path(type : Symbol, path)
       "/#{type}#{path}"
     end
+
+    class BeforeFilterAlreadyDefinedException < Exception
+    end
```

Comment:
You mean it would be a constant in the class ? Or maybe defined in the initializer ?